lib/before.js

```diff
@@ -99,8 +99,10 @@ module.exports = function(scope, cb) {
 	// Determine default values based on the
 	// available scope.
 
+	// Check if slash delimiter exists.
+	var slashIndex = scope.id.lastIndexOf('/');
 	_.defaults(scope, {
-		entity: _.str.capitalize(scope.id) + 'Controller',
+		entity: (slashIndex === -1 ? _.str.capitalize(scope.id) : _.str.splice(scope.id, slashIndex + 1, 1, scope.id.charAt(slashIndex + 1).toUpperCase())) + 'Controller',
```

Comment:
@1001v would you pull this out into a local variable and an if statement?  That'll make this code easier to come back to and maintain.  Also, and I realize this is out of the blue, so apologies, if you could please do it without the underscore.string dep, and relying on lodash 3.10.1, that'd be helpful.

Thanks!